### ISSUE TITLE
human plugin reports incorrect status on 92.1

### ISSUE THREAD

**Issue by nirGNU** [avocado-framework/avocado#5501]
**Describe the bug**
The stdout summary table through human plugin is showing incorrect status.
  
**Steps to reproduce**

Incorrect status with nrunner-status-server-uri 
 ```
avocado run  /bin/true --nrunner-status-server-uri 1.2.3.4:9000 --nrunner-status-server-disable-auto
JOB ID     : 16f11da720f0857914ad5c002c8b301b38dcf665
JOB LOG    : test/job-2022-10-29T16.12-16f11da/job.log
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0  ------------> Incorrect status
JOB TIME   : 1.34 s
```
Correct status without nrunner-status-server-uri 
```
avocado run  /bin/true
JOB ID     : 084cf853aac398b4ace70832543882b24514291a
JOB LOG    : /CTP/logs/test/job-2022-10-29T16.12-084cf85/job.log
 (1/1) /bin/true: STARTED
 (1/1) /bin/true: PASS (0.01 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1.35 s

```

**Expected behavior**
Expected a correct status.

**Current behavior**
What you are getting?

**System information (please complete the following information):**

- OS: CentOS 7
- Avocado version: Avocado 92.0
- Avocado installation method: pip

**Comment by clebergnu** [comment]
Hi @nirGNU,

Under 92.x you'll also need to add `--nrunner-status-server-listen` to have that working properly.

Let me know if that helps.

Thanks.

**Comment by nirGNU** [comment]
I have my status server running on a different host on which I am collecting logs, I do not want to use `--nrunner-status-server-listen` it will defeat the `--nrunner-status-server-uri` purpose 

With uri and listen both options: My server running on URI 
Same IP different Port
```
avocado run   /bin/true  --nrunner-status-server-uri 1.2.3.4:9000 --nrunner-status-server-disable-auto  --nrunner-status-server-listen 1.2.3.4:8000
JOB ID     : d544f1f84398cc01033e6921be79a39b3e4a55f1
JOB LOG    : test/job-2022-10-29T17.26-d544f1f/job.log
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 2 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 48.23 s

```
Same IP Same Port
```
OSError: [Errno 98] error while attempting to bind on address ('10.237.77.23', 9000): address already in use

```
Different IP different Port

```
avocado run   /bin/true  --nrunner-status-server-uri 1.2.3.4:10000 --nrunner-status-server-disable-auto  --nrunner-status-server-listen 5.6.7.8:9000
JOB ID     : 6b691a30f602b8b18243c41fd37e68b7bb81ae16
JOB LOG    : test/job-2022-10-29T17.32-6b691a3/job.log
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 2 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 48.15 s

```

**Comment by clebergnu** [comment]
The problem with sending the test statuses only to a different (remote) status server, is that the local status server won't see any test results.  Avocado relies on those status to be able to say which tests passed, which tests failed, etc. That's the reason for the lack of results, and the result that tests were not run (SKIP).

Now, it should be possible to send test (Tasks, actually as they called under the hood) statuses to more than one status server:

https://github.com/avocado-framework/avocado/blob/92lts/avocado/core/nrunner.py#L837

But that is not available to command line options at this point (famous last words).

Anyway, do you think we're on the same page here?  Also, I'd be very interested in knowing how you are actually running the status server on the other host.  Are you constructing the status summary and test results files at the remote server too?

**Comment by nirGNU** [comment]
It’s expensive for me to store logs locally and hence I’m grabbing those (historically by adding custom handlers in root logger or LOG_JOB & LOG_UI to be specific) and sending to remote server(real time), with nrunner I’m grabbing through URI and yes we are on same page here I’m constructing status log files in similar fashion of local.

**Comment by clebergnu** [comment]
@nirGNU this ended up being more than a simple fix, and with 103.0 LTS about to be released (we're only missing documentation related work), and your stated intention to use 103.0 LTS, I'm closing this issue for now.

**Comment by nirGNU** [comment]
Hi @clebergnu ,

That is fine, I have added a workaround as a plugin for this issue to get statuses reflected in human.
Having to have 103 LTS is my highest priority and my release is tied up with it. 
Its nice to know only documentation is remaining.

Thank you,
-Nir
